example/server/models.py

```diff
@@ -47,8 +47,15 @@ class PolarisUserTransaction(models.Model):
     this table serves to join the two entities.
     """
 
-    transaction = models.OneToOneField(Transaction, on_delete=models.CASCADE)
-    account = models.ForeignKey(PolarisStellarAccount, on_delete=models.CASCADE)
+    transaction_id = models.TextField(db_index=True)
```

Comment:
At the time of calling the `process_send_request()` integration, the `Transaction` object isn't created yet, since the integration could return a 400 response.

Reply:
So our reference anchor using Polaris saves the ID that _will_ be used to create the transaction, but not in a FK column which would check for the row's existence when inserted.